src/main/java/org/oscm/app/shell/business/actions/ProvisioningActions.java

```diff
@@ -54,6 +55,40 @@ public String executeScript(String instanceId, ProvisioningSettings settings,
         }
     }
 
```

Comment:
Please think of moving this method to other class (e.g. ScriptLogger or Script) as ProvisioningActions has other purpose.